opm/autodiff/BlackoilModelBase_impl.hpp

```diff
@@ -2454,15 +2517,6 @@ namespace detail {
 #endif
         }
 
-        // compute PAV and PORV for every regions.
```

Comment:
We probably need to keep this computation here as this is the only place where the sums over regions has global values.